<launch>

  <node pkg="camera1394stereo" type="camera1394stereo_node" name="camera1394stereo_node" output="screen" >
    <param name="video_mode" value="format7_mode3" />
    <param name="format7_color_coding" value="raw16" />
    <param name="bayer_pattern" value="bggr" />
    <param name="bayer_method" value="Simple" />
    <param name="stereo_method" value="Interlaced" />
    <param name="camera_info_url_left" value="" />
    <param name="camera_info_url_right" value="" />
  </node>
  
  <node pkg="dynamic_reconfigure" type="reconfigure_gui" name="reconfigure_gui" output="screen" />
  
  <node pkg="image_view" type="image_view" name="image_view_left">
    <remap from="image" to="stereo_camera/left/image_raw" />
  </node>

  <node pkg="image_view" type="image_view" name="image_view_right">
    <remap from="image" to="stereo_camera/right/image_raw" />
  </node>

  <!-- republish with theora compression -->
  <node pkg="image_transport" type="republish" name="theora"
        args="raw theora">
    <remap from="in" to="/stereo_camera/left/image_raw"/>
    <remap from="out" to="/stereo_camera/left/image_raw"/>
  </node>

</launch>
